cellfinder_napari/detect.py

```diff
@@ -194,4 +204,7 @@ def restore_defaults() -> None:
             if value is not None:  # ignore fields with no default
                 getattr(widget, name).value = value
 
+    # Insert progress bar before the run and reset buttons
+    widget.insert(-3, progress_bar)
```

Comment:
is `-3` a position?

Reply:
Yes

Reply:
could `-3` be made into a variable then, rather than a "magic number"?